components/TextFields/BUILD

```diff
@@ -115,6 +115,9 @@ mdc_snapshot_objc_library(
 
 mdc_snapshot_test(
     name = "snapshot_tests",
+    # Tests have sometimes taken more than 5 minutes to launch the simulator
+    # and execute
+    timeout = "long",
```

Comment:
Included here to help deflake, but will be merged separately in https://github.com/material-components/material-components-ios/pull/6337